datasets/nell/README.md

```diff
@@ -0,0 +1,234 @@
+certainties of belief are lower. The two sentences config extracts the
+CPL sentence patterns filled with the applicable 'best' literal string
+for the entities filled into the sentence patterns. And also provides
+sentences found using web searches containing the entities and
+relationships.
+
+There are roughly 21M entries for nell_belief_sentences, and 100M
+sentences for nell_candidate_sentences.
+
+From the NELL website:
+
+- **Research Goal**
+To build a never-ending machine learning system that acquires the ability to extract structured information from unstructured web pages. If successful, this will result in a knowledge base (i.e., a relational database) of structured information that mirrors the content of the Web. We call this system NELL (Never-Ending Language Learner).
+
+- **Approach**
+The inputs to NELL include (1) an initial ontology defining hundreds of categories (e.g., person, sportsTeam, fruit, emotion) and relations (e.g., playsOnTeam(athlete,sportsTeam), playsInstrument(musician,instrument)) that NELL is expected to read about, and (2) 10 to 15 seed examples of each category and relation.
+
+Given these inputs, plus a collection of 500 million web pages and access to the remainder of the web through search engine APIs, NELL runs 24 hours per day, continuously, to perform two ongoing tasks:
+
+Extract new instances of categories and relations. In other words, find noun phrases that represent new examples of the input categories (e.g., "Barack Obama" is a person and politician), and find pairs of noun phrases that correspond to instances of the input relations (e.g., the pair "Jason Giambi" and "Yankees" is an instance of the playsOnTeam relation). These new instances are added to the growing knowledge base of structured beliefs.
+Learn to read better than yesterday. NELL uses a variety of methods to extract beliefs from the web. These are retrained, using the growing knowledge base as a self-supervised collection of training examples. The result is a semi-supervised learning method that couples the training of hundreds of different extraction methods for a wide range of categories and relations. Much of NELL’s current success is due to its algorithm for coupling the simultaneous training of many extraction methods.
+
+For more information, see: http://rtw.ml.cmu.edu/rtw/resources
+
+### Languages
+en, and perhaps some others
+
+## Dataset Structure
+
+### Data Instances
+
+There are four configurations for the dataset: nell_belief, nell_candidate, nell_belief_sentences, nell_candidate_sentences. 
+
+nell_belief and nell_candidate defines:
+
+``
+{
+	'entity': ...,
+	'relation': ...,
+	'value': ...,
+	'iteration_of_promotion': ...,o
+	'probability': ...,
+	'entity_literal_strings': ...,
+	'value_literal_strings': ...
+	'best_entity_literal_string': ...
+	'best_value_literal_string': ...
+	'categories_for_entity': ...
+	'categories_for_value': ...
+	'candidate_source': ...
+}

```

Comment:
can you add real examples here ? (you ca crop some texts if they are too long)

You can get example by using `load_dataset` and taking the first sample with `dataset["train"][0]` for example

Reply:
Done